tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -174,6 +175,25 @@ Given(
   }
 )
 
+Given('the administrator has enabled shareapi exclude groups', function () {
+  return runOcc(
+    [
+      'config:app:set core shareapi_exclude_groups --value=yes'
```

Comment:
Does not this require to delete the set configs on `After` hooks? If it does, then, this PR needs to wait for #2269 to be merged.